controllers/persistentvolume/persistentvolume_controller_test.go

```diff
@@ -246,11 +247,83 @@ var _ = Describe("PersistentVolume Controller", func() {
 
 				Eventually(func() bool {
 					err := k8sClient.Get(ctx, pvLookupKey, createdPv)
+					if createdPv.Name == secondPersistentVolume || createdPv.Name == topologyPvWithArrayAddress {
+						Expect(createdPv.Spec.CSI.VolumeAttributes[util.PVMgmtAddrKey]).To(Equal("OS4xMTAuMTEuMjM="))
+					}
 					return err == nil
 				}, timeout, interval).Should(BeTrue())
 			}
 		})
 
+		It("should test the ensureStorageSystemLabel function successfully", func() {
```

Comment:
In order to test the logic of the PV reconcile function we need to verify the labels of the pv before calling `ensureStorageSystemLabel` manually.